### PR TITLE
[MM-15790] fixing an obscure bug for plugins - copy the array before modifying it

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- fixes an obscure bug for plugins. Don't modify the existing array, redux gets angry.

#### Ticket Link
[MM-15790](https://mattermost.atlassian.net/browse/MM-15790)

### CODE REVIEW

File: plugins/index.js [hunk 1]
```diff
@@ -119,7 +120,7 @@ export function removePlugin(manifest) {
     if (plugin && plugin.uninitialize) {
         plugin.uninitialize();
 
-    // Support the deprecated deinitialize callback from the plugins beta.
+        // Support the deprecated deinitialize callback from the plugins beta.
```
Comment (jwilander) [inline plugins/index.js]: Unrelated changes in this file?

Comment (cpoile) [reply plugins/index.js]: totally unrelated, I have no idea how that got in. Sorry, will fix.

Comment (cpoile) [reply plugins/index.js]: @jwilander better now
